applications/launchpad/docker_rig/config.toml

```diff
@@ -37,7 +37,8 @@ peer_seeds = [
 [base_node]
 network = "dibbler"
```

Comment:
This whole file needs to be replaced. I think let's merge it as is, and then we can update it